modules/http/http.c

```diff
@@ -424,11 +452,19 @@ http_dd_init(LogPipe *s)
 
   log_template_options_init(&self->template_options, cfg);
 
+  if (!(self->curl = curl_easy_init()))
+    {
+      msg_error("curl: cannot initialize libcurl", NULL);
+      return FALSE;
```

Comment:
hmm, we need to improve this (and other similar) error messages. You can get the location information about which source we are reporting the issues about with log_pipe_get_location_tag(), reporting the error message returned by libcurl will also help troubleshooting this message.